notebooks/source/truncated_distributions.ipynb

```diff
@@ -0,0 +1,1916 @@
+{
```

Comment:
<p>Perhaps move this explanation of <code>condition</code>  2 cells up, because that's the first place in the notebook where you use <code>condition</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>